website/client-old/js/controllers/menuCtrl.js

```diff
@@ -116,7 +116,7 @@ angular.module('habitrpg')
       };
 
       $scope.viewGroupApprovalNotification = function (notification, $index) {
-        User.readNotification(notification.id);
+        User.readNotifications([notification.id]);
```

Comment:
if we're marking only one notification as read, let's keep the old readNotification function